command/vsm_stats.go

```diff
@@ -0,0 +1,133 @@
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/rancher/go-rancher/client"
+)
+
+type Stats struct {
+	client.Resource
+	ReplicaCounter  int64 `json:"replicacounter"`
+	RevisionCounter int64 `json:"revisioncounter"`
```

Comment:
The last time I talked with @payes I felt these counters to be similar to raft indexes ?
If it is possible lets use standard conventions.